include/mata/alphabet.hh

```diff
@@ -169,10 +172,155 @@ namespace Mata {
             IntAlphabetSingleton &alphabet_instance;
         }; // class IntAlphabet.
 
-/**
- * An alphabet constructed 'on the fly'.
- * Should be use anytime the automata have a specific names for the symbols.
- */
+    /**
+     * Enumerated alphabet using a set of integers as symbols maintaining a set of specified symbols.
```

Comment:
I'd suggest in the comment some example of some EnumAlphabet...like, e.g. `Example of alphabet = {"1", "2", "3", ... }`.

Reply:
Great idea. Will do.